components/collapsible/collapsible.vue

```diff
@@ -105,7 +103,7 @@ export default {
      */
     anchorText: {
       type: String,
-      default: null,
+      default: 'Button',
```

Comment:
This was required for `axe-core accessibility`

_Personal Opinion_ I understand this as mandatory, we should have anchor with text as a default value.

Before it was added by the slot, but we removed the slot override to show the default button instead.

Here we have two options, give the text value by props, or just adding it as a default value.

Reply:
I think the correct way to fix this is with a custom validator, that validates for this:

> anchorText and anchor slot content cannot both be falsy

You can see an example in combobox.vue, `validateEmptyListProps()`

Reply:
Completely agree :)